clients/vault/src/oracle/testing_utils.rs

```diff
@@ -43,9 +43,16 @@ fn stellar_relay_config_abs_path(
 		.expect("should be able to extract config")
 }
 
-pub fn get_test_secret_key(is_mainnet: bool) -> String {
-	let file_name = if is_mainnet { "mainnet" } else { "testnet" };
-	let path = format!("./resources/secretkey/stellar_secretkey_{file_name}");
+pub fn get_mainnet_secret_key() -> String {
+	let path = "./resources/secretkey/stellar_secretkey_mainnet";
+	std::fs::read_to_string(path).expect("should return a string")
+}
+
+/// return a secret key for testnet with 2 choices; one has xlm, the other does not
```

Comment:
again, this is to address 
https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/vault/tests/helper/mod.rs#L33

Reply:
I think for these functions, we also need two different secret keys for the mainnet tests. I think we need one for the 'source' account and another one for the 'destination' at least in the integration tests.